core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -621,6 +621,26 @@ class HttpGoogleServicesDAO(val clientSecrets: GoogleClientSecrets,
         .unsafeToFuture()
     }
 
+  def testSAGoogleBucketGetLocation(googleProject: GoogleProject, bucketName: GcsBucketName, saKey: String)(implicit
+    executionContext: ExecutionContext
+  ): Future[Option[String]] = {
+    implicit val async = IO.asyncForIO
+    val credentials = ServiceAccountCredentials.fromStream(new ByteArrayInputStream(saKey.getBytes))
+    val storageServiceResource = GoogleStorageService.fromCredentials(credentials)
+    storageServiceResource
+      .use { storageService =>
+        storageService.getBucket(googleProject, bucketName)
```

Comment:
You need to provide a `BucketGetOption` with the `userProject` set if it's requester pays or this will fail

Reply:
How would we know which user-project to use? Instead, I'm thinking that we do what the UI does [here](https://github.com/DataBiosphere/terra-ui/blob/31ed64d4f3f3261cd69aeb1d4a987f1ea7e16381/src/libs/ajax/ajax-common.ts#L100) and just exclude requester-pays errors from triggering the FastPass functionality.